baxter_collaboration_lib/test/test_ros_thread.cpp

```diff
@@ -0,0 +1,37 @@
+#include <gtest/gtest.h>
+#include "robot_utils/ros_thread.h"
+//#include <pthread.h>
+//#include <typeinfo>
+#include "robot_utils/ros_thread_obj.h"
+
+
+
+class ROSThreadObject: public ROSThread
```

Comment:
What's the difference between this class and this one: https://github.com/ScazLab/baxter_collaboration/blob/master/baxter_collaboration_lib/include/robot_utils/ros_thread_obj.h ?

They seem different, I would call yours `ROSThreadTester`.